Tuto-GUDHI-ripsnet.ipynb

```diff
@@ -0,0 +1,1213 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from tqdm import tqdm 
</pre><p>This implies to add <code>tqdm</code> in <code>requirements.txt</code> file </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Thanks, done.